specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/CommonDefinitions.json

```diff
@@ -1205,6 +1223,10 @@
                 "modelAsString": true
               }
             },
+            "keyVaultProperties": {
```

Comment:
Same here as shared in the above comments.

Reply:
Let's use `certificateKeyVaultProperties`

Reply:
Let's also update the description: "Certificate stored in Azure Key Vault"